### PR TITLE
Add manual expectations list for an inject

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Related issues

* https://github.com/OpenEx-Platform/openex/issues/410

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: openex-front/src/admin/components/exercises/injects/expectations/ExpectationManualForm.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        variant="standard"
+        fullWidth={true}
+        label={t('Name')}
+        error={!!errors.expectation_name}
+        helperText={
+          errors.expectation_name && errors.expectation_name?.message
+        }
+        inputProps={register('expectation_name')}
+      />
+      <MuiTextField
```
Comment (RomuDeuxfois) [inline openex-front/src/admin/components/exercises/injects/expectations/ExpectationManualForm.tsx:65]: Use multilie field

File: openex-front/src/components/Color.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const colorStyles = {
```
Comment (guillaumejparis) [inline openex-front/src/components/Color.ts:1]: color not in the theme ?

**Comment by guillaumejparis** [comment]
- [ ] sorting do not work on expectations array

**Comment by RomuDeuxfois** [comment]
Migration for Content : SMS & MAIL

**Review by guillaumejparis — APPROVED** [review]

**Pull request merged**